src/main/resources/hudson/plugins/active_directory/ActiveDirectorySecurityRealm/help-cacheTtl.html

```diff
@@ -0,0 +1,8 @@
+<div>
+    Some LDAP servers may be slow, or rate limit client requests. In such cases enabling caching may improve performance
```

Comment:
this is generic to the cache - so can this be pulled into the outer help - and leave this just to the specific of the size?